flow/project.py

```diff
@@ -3574,7 +3572,12 @@ class MyProject(FlowProject):
 
         parser_status = subparsers.add_parser(
             'status',
-            parents=[base_parser])
+            parents=[base_parser],
+            help="You can now specify the parallelization of the status command "
+                 "by seting the the flow.status_parallelization config "
```

Comment:
```suggestion
            help="You can specify the parallelization of the status command "
                 "by setting the the flow.status_parallelization config "
```
Remove the "now" (this is not a changelog) but describes the *status quo* and there was a typo in the second line.

Also, I think we should mention that "thread" is the default value.